.github/workflows/azure-static-web-apps-purple-plant-006657c1e.yml

```diff
@@ -2,29 +2,27 @@ name: Azure Static Web Apps CI/CD
 
 on:
   push:
-    branches:
-      - main
-    paths:
-      - "wordle-web/**"
   pull_request:
+    types: [opened, synchronize, reopened, closed]
+    branches:
+      - Assignment2

```

Comment:
Might want to change this to Assignment3 branch

Reply:
My own partner and I chose to use the main branch for this too - you can develop in main and the assignment branches can just act as "snapshots" of the main, it can be a bit more convenient.

Reply:
Nathan that's actually a really good idea haha. I think I'll do that in the future. With how I'm currently doing it, Hex's suggestion makes more sense though. Thanks to the both of you!